components/server/src/user/user-service.ts

```diff
@@ -231,12 +294,18 @@ export class UserService {
     async getWorkspaceUsageAttributionId(user: User, projectId?: string): Promise<string | undefined> {
```

Comment:
nit, and out-of-scope here: It feels like this method (and it's descendants) do not necessarily belong into UserService, but more into `usage/Attribution` or so. But that's sth for later. :upside_down_face: